pep-0726.rst

```diff
@@ -43,6 +43,8 @@ For example
 
 .. code:: python
 
+   # mlib.py
```

Comment:
Is this comment needed? I'm not sure what it signifies.

Reply:
I think it is.  Else "import mlib" statement in the console session example could be slightly cryptic.

Reply:
```suggestion
   # mplib.py
```